### PR TITLE
CVF-7168 amend support for python/nameko

### PR DESCRIPTION
**Pull request by tomviner**
- Drop support for Python < 3.6.
- Add tests for new Python and Nameko 2.x versions.
- Convert tests to github-actions, as travis free tier is no longer sufficient 

Todo:
- [x] publish to public pypi, once approved

### CODE REVIEW

File: .github/workflows/tox-tests.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox~=3.28 'tox-gh-actions<3'
```
Comment (heindsight) [inline .github/workflows/tox-tests.yml:25]: Why do we need to pin these?

Comment (tomviner) [reply .github/workflows/tox-tests.yml:25]: Tox v4 is a complete rewrite of the package and requires Python 3.7. 

Also I didn't want to have to check all these https://tox.wiki/en/latest/upgrading.html

The `tox-gh-actions`  restriction, is just for tox 3 compat

Comment (heindsight) [reply .github/workflows/tox-tests.yml:25]: Fair enough :+1:

**Review by heindsight — APPROVED** [review]

**Review by juliotrigo — APPROVED** [review]
👍

Same question about the Python versions:https://github.com/sohonetlabs/nameko-statsd/pull/17

**Comment by tomviner** [comment]
> Same question about the Python versions:[sohonetlabs/nameko-statsd#17](https://github.com/sohonetlabs/nameko-statsd/pull/17)

Let's try

**Pull request merged**